airflow/providers/cncf/kubernetes/executors/kubernetes_executor_utils.py

```diff
@@ -230,7 +230,6 @@ def process_status(
                 self.kube_config.worker_pod_pending_fatal_container_state_reasons
                 and "status" in event["raw_object"]
             ):
-                self.log.info("Event: %s Pending, annotations: %s", pod_name, annotations_string)
```

Comment:
why this line needs to be moved down?

Reply:
I forgot to add an else statement to the for loop. When there are no fatal reasons, the code path should be to print out the pod pending log.